src/components/panel/index.jsx

```diff
@@ -43,30 +46,63 @@ export default class Panel extends Component {
         activeStory: story
     });
     this.fetchComments(kind, story);
+    this.setState({ userComment: '' });
   }
   verifyUser() {
-    const userNickName = localStorage.getItem('blabbr_userNickName');
+    const userName = localStorage.getItem('blabbr_userName');
     const userEmail = localStorage.getItem('blabbr_userEmail');
-    userNickName && userEmail && this.setState({ user: { userNickName,  userEmail, isUserAuthenticated: true }});
+    userName && userEmail && this.setState({ user: { userName,  userEmail, isUserAuthenticated: true }});
   }
-  registerUser(nickname, email) {
+  registerUser(username, email) {
     const { user } = this.state;
-    localStorage.setItem('blabbr_userNickName', nickname);
+    localStorage.setItem('blabbr_userName', username);
     localStorage.setItem('blabbr_userEmail', email);
     this.setState({ user: Object.assign(user, { isUserAuthenticated: true })});
   }
-  onUserNickNameChange(e) {
+  onUserNameChange(e) {
     const { user } = this.state;
-    this.setState({ user: Object.assign(user, { userNickName: e.target.value })});
+    this.setState({ user: Object.assign(user, { userName: e.target.value })});
   }
   onUserEmailChange(e) {
     const { user } = this.state;
     this.setState({ user: Object.assign(user, { userEmail: e.target.value })});
   }
   onRegisterSubmit(e) {
-    const { user: { userNickName, userEmail } } =  this.state;
+    const { user: { userName, userEmail } } =  this.state;
     e.preventDefault();
-    this.registerUser(userNickName, userEmail);
+    this.registerUser(userName, userEmail);
+  }
+  onUserCommentChange(e) {
+    this.setState({ userComment: e.target.value });
+  }
+  onCommentSubmit(e) {
+    const { userComment } = this.state;
+    e.preventDefault();
+    this.postComment(userComment);
+  }
+  postComment(userComment) {
+    const {
+      user: { userName, userEmail },
+      activeComponent,
+      activeStory,
+      activeVersion,
+      comments,
+    } = this.state;
+
+    postComment({
+      userComment,
+      userName,
+      userEmail,
+      component: activeComponent,
+      story: activeStory,
+      version: activeVersion,
+    })
+      .then((data) => {
```

Comment:
This will be done once I'm finished the 'listener' integration, but keep in for now. e.g. comment gets added -> listener on db gets fired -> state gets changed -> react renders new comments list